### DISCUSSION TITLE
How does Censorship work?

### DISCUSSION THREAD

**Discussion by ericaVoss** [EHDEN/CatalogueExport#36] [Q&A] [answered]
I know that ACHILLES doesn't show records that have a cell size count of 5 or less.  The minimum cell count in CatalogExport is 100.  How that the censorship and rounding work in CatalogExport?

If I have 6 rows, would it be excluded or rounded to 100?

**Comment by MaximMoinat** [comment, marked as answer]
They will be rounded to 100.

Whether it is excluded depends on how the `smallCellCount` parameter of `catalogueExport` function is set.
If the `smallCellCount is set to 100, then all counts from 1 to 100 are excluded in the results

Note that by default the `smallCellCount` is set to 5, meaning that all counts are included (albeit rounded up to 100 at a minimum).

**Reply by MaximMoinat** [reply]
I opened an issue for that last note: https://github.com/EHDEN/CatalogueExport/issues/37

**Reply by MaximMoinat** [reply]
It should also be noted that analyses that are not based on concept_ids, are not rounded to nearest 100:
 - 0,1,2,3,101,102,108,109,110,111,112,113,117 (person counts by gender, age, obs_period length, etc.)
 - 220, 420, 502, 720, 820, 920, 1020, 1820, 2120 (count by month)
